test/hotspot/jtreg/runtime/Nestmates/membership/TestNestHostErrorWithMultiThread.java

```diff
@@ -48,12 +48,12 @@ public int foo() {
 
 public class TestNestHostErrorWithMultiThread {
 
-  public static void main(String args[]) {
+  public static void main(String args[]) throws Throwable {
```

Comment:
"throws Throwable" is only necessary if you want unlisted checked exceptions to propagate.

Reply:
Thank you for your review. `throws Throwable` will be removed in my next commit.